operators/volume_loader/volume.hpp

```diff
@@ -54,6 +54,10 @@ class Volume {
   std::array<bool, 3> flip_axes_{false, false, false};
   /// frame duration
   std::chrono::duration<float> frame_duration_;
+  /// space origin
+  std::array<double, 3> space_origin_{0.0, 0.0, 0.0};
+  /// space directions
+  std::vector<std::array<double, 3>> space_directions_;
```

Comment:
It seems these are not used but the volume renderer. Do you plan to parse these in the renderer?

Reply:
I need your help with that as I have no experience with it.